src/xtgeo/surface/_regsurf_ijxyz_parser.py

```diff
@@ -168,7 +173,13 @@ def _parse_arrays(self) -> None:
     def _map_on_template(self) -> None:
         """An existing RegularSurface or Cube forms the geometrical template."""
         logger.debug("Parse ijxyz with template")
-        if not isinstance(self.template, (xtgeo.RegularSurface, xtgeo.Cube)):
+
+        # TODO: Remove these when moved to xtgeo.io
+        from xtgeo.cube.cube1 import Cube
+
+        from .regular_surface import RegularSurface
+
+        if not isinstance(self.template, (RegularSurface, Cube)):
```

Comment:
This is ugly on my part. This class will end up pulled into `xtgeo.io`, however, since it is strictly parsing and can be further addressed there.